docs/src/references/tedge-config-dir.md

```diff
@@ -0,0 +1,73 @@
+# `tedge --config-dir`
```

Comment:
I think this title is not clear. You can discuss the title and location with Didier. I assume under "Command Line Reference" is the appropriate location, however, not really sure. 

```suggestion
# The `--config-dir` option
```